assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/data/useDataReducer.ts

```diff
@@ -13,6 +13,7 @@ export const initialState: DataState = {
 const useDataReducer = (initializer: StateInitializer): DataStateReducer => {
 	const dataReducer: DataStateReducer = (state, action) => {
 		const { type, id, index, fieldValues, ticketPrice, price, prices } = action;
+		let isTaxable: boolean;
```

Comment:
what was the reason it was pulled at a higher level?

Reply:
because it's use in many switch cases.